### ISSUE TITLE
Typo extra-annos and extra_annos

### ISSUE THREAD

**Issue by incardon** [bihealth/varfish-server#501]
**Describe the bug**
Import extra_annos table fail with the background database

**To Reproduce**
Download the background database and try to import extra_annos

**Expected behavior**
Import extra annos can be imported

**Desktop (please complete the following information):**
All

**Additional context**
It could be that this has been already fixed. But it look like there is a typo in the import_versions.tsv of the varfish-server-background-db-20201006.tar.gz. The file I downloaded weeks ago (with the GRCh37 bug) report

extra-annos

like in the documentation at page:

https://varfish-server.readthedocs.io/en/latest/developer_database.html

Try to import the table result in a failure:

Disabling autovacuum on all tables...
Table group extra-annos is no registered table group.
Enabling autovacuum on all tables...

In debug mode it turns out that the failing line is at 233 of import_tables.py

```
                for import_info in import_infos:
                    if import_info["table_group"] in TABLES[import_info["build"]]:
                        pool.apply_async(self._handle_import_try_catch, (import_info, options))
                    else:
                        self.stderr.write(
                            "Table group {} is no registered table group.".format(
                                import_info["table_group"]
                            )
                        )
```

In short the code is expecting "extra_annos" but the import_info["table_group"] has "extra-annos" from import_versions.tsv, leading to the error

import_table.py line 72 

    "ensembltorefseq": (EnsemblToRefseq,),
    "extra_annos": (ExtraAnno, ExtraAnnoField),
    "gnomAD_constraints": (GnomadConstraints,), 

(I workaround fixing the name manually in the backgrouns database)

**Comment by holtgrewe** [comment]
This looks like an issue with the documentation not being updated everywhere.

The following has correct and tested paths

https://varfish-server.readthedocs.io/en/latest/developer_data_builds.html

Does changing the download URL fix the issue?

**Comment by incardon** [comment]
wget https://file-public.bihealth.org/transient/varfish/anthenea/varfish-server-background-db-20201006.tar.gz

End in a 404

My guess for the new file would have been

wget https://file-public.bihealth.org/transient/varfish/anthenea/varfish-server-background-db-.20210728.tar.gz

also end in a 404

which is the name of the new file ?

**Comment by holtgrewe** [comment]
It's the "b" file.


https://file-public.bihealth.org/transient/varfish/anthenea/


--
Dr. Manuel Holtgrewe, Dipl.-Inform.
Bioinformatician
Core Unit Bioinformatics – CUBI
Berlin Institute of Health / Max Delbrück Center for Molecular Medicine in the Helmholtz Association / Charité – Universitätsmedizin Berlin

Visiting Address: Invalidenstr. 80, 3rd Floor, Room 03 028, 10117 Berlin
Postal Address: Chariteplatz 1, 10117 Berlin

E-Mail: ***@***.***
Phone: +49 30 450 543 607
Fax: +49 30 450 7 543 901
Web: cubi.bihealth.org  www.bihealth.org  www.mdc-berlin.de  www.charite.de
________________________________
From: incardon ***@***.***>
Sent: Wednesday, May 11, 2022 9:17:53 AM
To: bihealth/varfish-server
Cc: Holtgrewe, Manuel; Comment
Subject: [ext] Re: [bihealth/varfish-server] Typo extra-annos and extra_annos (Issue #501)


wget https://file-public.bihealth.org/transient/varfish/anthenea/varfish-server-background-db-20201006.tar.gz

End in a 404

My guess for the new file would have been

wget https://file-public.bihealth.org/transient/varfish/anthenea/varfish-server-background-db-.20210728.tar.gz

also end in a 404

which is the name of the new file ?

**Comment by incardon** [comment]
The new one looks good
